.angular-cli.json

```diff
@@ -7,7 +7,8 @@
       "root": "src",
       "outDir": "dist",
       "assets": [
-				"assets"
+				"assets",
+        "manifest.json"
```

Comment:
Indentation issue here. Please take a look.

Reply:
@singhpratyush I have checked in two editors atom and sublime.It looks good but I don't know why it is showing like this. I can do it by the online editor but then I will not be able to squash commits.

Reply:
![image](https://user-images.githubusercontent.com/24358501/35180579-7c92ef90-fdd9-11e7-86e7-9512ca718397.png)
![image](https://user-images.githubusercontent.com/24358501/35180582-8b22569a-fdd9-11e7-9e2f-ec5bed6a3cd7.png)

Reply:
The problem occurs because the file is a mixture of both tabs and spaces. Please make it consistent and add a configuration in [`[.editorconfig]`](https://github.com/fossasia/loklak_search/blob/master/.editorconfig).